language-support/hs/bindings/BUILD.bazel

```diff
@@ -9,6 +9,7 @@ da_haskell_library(
     name = "hs-ledger",
     srcs = glob(["src/**/*.hs"]),
     hackage_deps = [
+        "MissingH",
```

Comment:
Adding a dep on `MissingH` just for `strip` seems a bit excessive. If you do want to strip, I would prefer to use `Text.strip` or just inline an implementation using `takeWhile`.